src/NHibernate/Cache/ReadWriteCache.cs

```diff
@@ -136,6 +146,45 @@ public object Get(CacheKey key, long txTimestamp)
 			}
 		}
 
+		public object[] GetMultiple(CacheKey[] keys, long txTimestamp)
```

Comment:
Maybe rename to just `Get` or to `GetMany`?

Reply:
Why not, both are fine for me. Which would you prefer?

Reply:
I would vote for `GetMany`. It is more immediate to understand its purpose with this nam, compared to `Get` alone which would require interpreting its arguments/return value.

Reply:
`GetMany` it is.

Reply:
Done.